src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -102,6 +102,17 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-locals
         profiles = os.path.expanduser("~/.dbt/profiles.yml")
 
     file_path = Path(file)
+
+    if "MANAGER_URL" not in ctx.obj and disallow_edits:
+        warnings.warn(
+            (
+                "The managed externally feature was only introduced in Superset v1.5."
+                "Make sure you are running a compatible version."
+            ),
+            category=UserWarning,
+            stacklevel=2,
+        )
```

Comment:
Thought about only including `is_managed_externally` to the update payloads in case `--disallow-edits` flag was used, but then I realized that if a user wanted to trigger a new sync without this flag to set it to `False`, that wouldn't work. So I only added this warning which in addition to the error faced for legacy instances should be pretty straightforward (the error would actually happens even if `--disallow-edits` isn't included).